<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseNoticeForm</class>
 <widget class="QDialog" name="ExerciseNoticeForm">
  <property name="windowTitle">
   <string>Уведомление об учении</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelExerciseName">
     <property name="text"><string>Название учения:</string></property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="exerciseName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelStartTime">
     <property name="text"><string>Дата и время начала:</string></property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="startTime"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelRegion">
     <property name="text"><string>Район проведения:</string></property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="region"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelNotes">
     <property name="text"><string>Примечание:</string></property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTextEdit" name="notes"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="buttonSend">
     <property name="text"><string>Отправить уведомление</string></property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>